.travis.yml

```diff
@@ -22,6 +22,11 @@ addons:
       - cmake
       - cmake-data
 
+env:
+  global:
+  - TRAVIS_CI=1
```

Comment:
Why is this necessary? Travis sets plenty environment anyway, why can't that be used?